src/node_native_module_env.cc

```diff
@@ -85,23 +84,45 @@ void NativeModuleEnv::GetCacheUsage(const FunctionCallbackInfo<Value>& args) {
   Isolate* isolate = env->isolate();
   Local<Context> context = env->context();
   Local<Object> result = Object::New(isolate);
-  result
+
+  Local<Value> native_modules_with_cache_js;
+  Local<Value> native_modules_without_cache_js;
+  Local<Value> native_modules_in_snapshot_js;
+  if (!ToV8Value(context, env->native_modules_with_cache)
+      .ToLocal(&native_modules_with_cache_js)) {
```

Comment:
Not sure if allowed by the linter, but the `.` should not align with the `!` IMO.

```suggestion
  if (!ToV8Value(context, env->native_modules_with_cache)
       .ToLocal(&native_modules_with_cache_js)) {
```

Reply:
> Not sure if allowed by the linter

The c++ linter doesn't care about indentations for now.

> but the `.` should not align with the `!` IMO.

That would be 5 spaces instead of what's suggested in the style guide, i.e., 4 - https://github.com/nodejs/node/blob/HEAD/doc/contributing/cpp-style-guide.md#4-spaces-of-indentation-for-statement-continuations. Should the doc be updated with more info about this special case?

Reply:
I think we’re following the 4-spaces rule pretty universally, and I wouldn’t change these kinds of cases to be special in that regard.